drone-ci/README.md

```diff
@@ -10,11 +10,20 @@ The setup requires Terraform (it has been tested with version 0.7.x) and Ansible
 
 ```
 module "drone" {
-  source = "github.com/nearform/labs-devops//drone-ci/drone"
-  keypair_key = "<your-public-key>"
-  private_key_path = "~/.ssh/id_rsa"
-  drone_vpc_id = "12345"
-  drone_subnet_id = "678910"
+  source = "github.com/nearform/labs-devops/drone-ci/drone"
```

Comment:
I think double slash is necessary to split the sub folder from the repo name as far as I can remember from the documentation.